lib.php

```diff
@@ -1113,6 +1114,9 @@ function bigbluebuttonbn_check_updates_since(cm_info $cm, $from, $filter = array
 function mod_bigbluebuttonbn_get_fontawesome_icon_map() {
     return [
         'mod_bigbluebuttonbn:icon' => 'icon-bigbluebutton',
+        'mod_bigbluebuttonbn:t/copy' => 'fa-copy',
```

Comment:
Is it  my imagination or this POR includes some other customizations you have?

Reply:
The Copy is used for a small button for copying the link or guest password. I checked the code, there is nothing except for the guest access feature